Formula/texlab.rb

```diff
@@ -0,0 +1,27 @@
+require "language/node"
+
+class Texlab < Formula
+  desc "Implementation of the Language Server Protocol for LaTeX"
+  homepage "https://texlab.netlify.com/"
+  url "https://github.com/latex-lsp/texlab/archive/v1.7.0.tar.gz"
+  sha256 "aa36f3f23656da0d58357fe044d2f60bc01a99bbaa2481132231f4791ff82d17"
+  head "https://github.com/latex-lsp/texlab.git"
+
+  depends_on "node" => :build
```

Comment:
Does it package node in the binary? Or does this just resolve dependencies?

Reply:
This is just used to resolve dependencies, node is not bundled.